src/Styleguide/Pages/Artist/index.tsx

```diff
@@ -16,7 +19,46 @@ import { Overview } from "./Routes/Overview"
 import { RelatedArtistsRoute } from "./Routes/RelatedArtists"
 import { ShowsRoute } from "./Routes/Shows"
 
-const Tab = PreloadLink
+const Navigation = ({ children }) => {
+  return (
+    <React.Fragment>
+      <RouterTabs>
+        <PreloadLink to="/" exact>
+          <Sans size="3t" weight="medium">
+            Overview
+          </Sans>
+        </PreloadLink>
+        <PreloadLink to="/cv">
+          <Sans size="3t" weight="medium">
+            CV
+          </Sans>
+        </PreloadLink>
+        <PreloadLink to="/articles">
+          <Sans size="3t" weight="medium">
+            Articles
+          </Sans>
+        </PreloadLink>
+        <PreloadLink to="/shows">
+          <Sans size="3t" weight="medium">
+            Shows
+          </Sans>
+        </PreloadLink>
+        <PreloadLink to="/auction-results">
+          <Sans size="3t" weight="medium">
+            Auction results
+          </Sans>
+        </PreloadLink>
+        <PreloadLink to="/related-artists">
+          <Sans size="3t" weight="medium">
+            Related artists
+          </Sans>
+        </PreloadLink>
+      </RouterTabs>
```

Comment:
I'll move this into a `<RouterTabs>` component in a follow-up PR that follows same API as `<Tabs>`.